models/userStatus.js

```diff
@@ -0,0 +1,115 @@
+const { userState } = require("../constants/userStatus");
```

Comment:
Why is the return part outside the catch block in all model functions?

Reply:
I am getting the error message `error  Expected to return a value at the end of async arrow function  consistent-return` while running the command `yarn lint-fix`. Hence I added the return outside the try-catch block.